src/coldcard.js

```diff
@@ -211,26 +220,17 @@ export class ColdcardExportPublicKey extends ColdcardFileParser {
   }
 
   messages() {
-    const messages = super.messages();
-    messages.unshift({
-      state: PENDING,
-      level: INFO,
-      code: "coldcard.export",
-      text: "Go to Settings > Multisig Wallets > Export XPUB",
-    });
-    return messages;
+    return super.messages();
   }
 
   parse(xpubJSONFile) {
     const result = super.parse(xpubJSONFile);
```

Comment:
You could simplify this method even more by doing `const { p2sh, rootFingerprint } = super.parse(xpubJSONFile)`. No big deal, but just thought about it since then line 233 doesn't need to change.